### PR TITLE
[Merged by Bors] - doc: replace `variables`, `universes`' syntax in doc comments

### PR DESCRIPTION
**Pull request by grunweg**
It's deprecated in favour of `variable`; likely a leftover from the port.
Also replace `universes`, which is invalid now.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Category/Basic.lean [hunk 1]
```diff
@@ -43,12 +43,12 @@ Typically any concrete example will either be a `SmallCategory`, where `v = u`,
 which can be introduced as
 ```
 universes u
```
Comment (eric-wieser) [inline Mathlib/CategoryTheory/Category/Basic.lean:45]:
```suggestion
universe u
```

**Review by eric-wieser — APPROVED** [review]
bors d+

It looks like you forgot to push the `universe` changes

File: Mathlib/CategoryTheory/Category/Basic.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/CategoryTheory/Category/Basic.lean:45]: Good catch, thank you.

**Comment by grunweg** [comment]
Oh, indeed - thanks for pointing this out. I have added them now.
bors r+

**Pull request closed without merging**